app/controllers/podcasts_controller.rb

```diff
@@ -17,7 +17,12 @@ def index
   def show
     authorize @podcast
 
-    @recently_published = @podcast.episodes.published.dropdate_desc.limit(3)
+    @recently_published_episodes = @podcast.episodes.published.dropdate_desc.limit(4)
+    @trend_episodes = @podcast.default_feed.episodes.published.dropdate_desc.where.not(first_rss_published_at: nil).offset(1).limit(4)
```

Comment:
here i am pulling the episodes to be compared against in the trend calculation. offset by 1 because i don't want the most recent one.